src/raster/r.forcircular/r.forcircular

```diff
@@ -0,0 +1,2039 @@
+#!/usr/bin/env python3
+# -- coding: utf-8 --
```

Comment:
You can remove this "coding utf-8" line, it's not needed with Python 3 (see also #497).

Reply:
Done